lib/src/page_list_viewport_gestures.dart

```diff
@@ -288,17 +303,35 @@ class _PageListViewportGesturesState extends State<PageListViewportGestures> wit
   }
 }
 
-class DeprecatedPanAndScaleVelocityTracker {
-  /// The maximum velocity a gesture can have and still be considered a viewport
-  /// re-adjustment. Used in tandem with the [kViewportReAdjustmentMinTranslationDistance]
-  /// to determine if the user is re-adjusting the viewport. Viewport re-adjustments
-  /// result in the momentum simulation to be aborted.
-  static double kViewportReAdjustmentMaxVelocity = 600;
+// statically defined class with double constant values which parametrize
```

Comment:
This should be a Dart Doc, not just a code comment. Also, it should follow the grammatical and structural standards for Dart Docs.

Reply:
Got it.